packages/server/modules/workspaces/graph/resolvers/workspaces.ts

```diff
@@ -314,9 +315,18 @@ export = FF_WORKSPACES_MODULE_ENABLED
 
           return await getPendingTeam({ workspaceId: parent.id })
         },
-        projects: async () => {
-          // Get projects in workspace
-          throw new WorkspacesNotYetImplementedError()
+        projects: async (parent, args) => {
+          const getWorkspaceProjects = getWorkspaceProjectsFactory({ getStreams })
+          return await getWorkspaceProjects(
+            {
+              workspaceId: parent.id
+            },
+            {
+              limit: args.limit ?? 25,
```

Comment:
`||` is in most cases better than `??` cause it'll use your fallback value not just when the 1st operand is `null` but when its generally falsy in any way, which is usually what you're after. Surely you want the fallback value irregardless of whether cursor is null, or `false` or an empty string or 0?

Reply:
back and forth on this one but will prefer `||` going forwards, this is a good reason